From 533cf5cfdfe047d2689e33c5e624325c3d9ffe38 Mon Sep 17 00:00:00 2001
From: Jia Jia <angersax@sina.com>
Date: Sun, 30 Jun 2024 10:42:20 +0800
Subject: [PATCH] Add test files 11:13

Change-Id: Id021b6659beb19b84437191d294b72d9c294df3b
---

diff --git a/lintshell/test.sh b/lintshell/test.sh
new file mode 100755
index 0000000..bb54fe5
--- /dev/null
+++ b/lintshell/test.sh
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "Hello Shell!"
